cudax/examples/stf/binary_fhe_stackable.cu

```diff
@@ -0,0 +1,203 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of CUDASTF in CUDA C++ Core Libraries,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2022-2024 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+/**
+ * @file
+ * @brief A toy example to illustrate how we can compose logical operations
+ *        over encrypted data
+ */
+
+#include "cuda/experimental/__stf/utility/stackable_ctx.cuh"
+#include "cuda/experimental/stf.cuh"
+
+using namespace cuda::experimental::stf;
+
+class ciphertext;
+
+class plaintext {
+public:
+    plaintext(const stackable_ctx& ctx) : ctx(ctx) {}
+
+    plaintext(stackable_ctx& ctx, std::vector<char> v) : values(v), ctx(ctx) {
+        l = ctx.logical_data(&values[0], values.size());
+    }
```

Comment:
This is creating a copy rather than a move. also we should rather use `values.data()` instead of `&values[0]`
```suggestion
    plaintext(stackable_ctx& ctx, ::std::vector<char> v) : values(::cuda::std::move(v)), ctx(ctx) {
        l = ctx.logical_data(values.data(), values.size());
    }
```